cmd/webhook/app/server.go

```diff
@@ -95,7 +88,7 @@ func (a *admissionHandler) serve(w io.Writer, r *http.Request, admit admitFunc)
 func (a *admissionHandler) mutateResource(ar v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 	klog.V(2).Info("mutating custom resource")
 	raw := ar.Request.Object.Raw
-	crd := apiextensionsv1beta1.CustomResourceDefinition{}
+	crd := apiextensionsv1.CustomResourceDefinition{}
```

Comment:
I am not sure why we use CustomResourceDefinition here. It seems to be an old bug.

Reply:
let's update this part as a followup